### ISSUE TITLE
Only first epoch runs longer, rest all the epochs run within 3-4 seconds

### ISSUE THREAD

**Issue by vikas95** [martbert/decomp_attn_keras#1]
Hi,

Thanks for sharing the code, it is helpful and a clean code. I was trying to run this Keras version of Decompatt model in order to compare with other versions of the model. 

I noticed that only the first epoch runs for long time (~1 hour) and rest all the epochs are extremely fast.
I am suspecting that it is training for just the 1st epoch and just evaluating on rest of the epochs. 

I have changed labels settings for just 2 labels (for question matching task) instead of 3 labels. This is the only change I have done, and I cannot find how to fix the training part for 2nd epoch onwards. 

It will be very helpful if you can suggest something.

Thanks, looking forward for your response,

-Vikas
